src/storage.js

```diff
@@ -471,4 +570,6 @@ export class LocalStorageStore {
   }
 }
 
-export default new LocalStorageStore();
+const STORE = new LocalStorageStore();
+
+export default STORE;
```

Comment:
question: Do you think this PR would be the most adequate scope to add unit tests for the Desktop Wallet storage? Or should we add those in the future?